src/components/folders/FolderItem.jsx

```diff
@@ -1,125 +1,27 @@
-          <chakra.button
-            className={`${elementStyles.card} ${contentStyles.card} ${elementStyles.folderItem}`}
-          >
-            <div className={contentStyles.contentContainerFolderRow}>
-              {type === "file" ? (
-                <i
-                  className={`bx bxs-file-blank ${elementStyles.folderItemIcon}`}
-                />
-              ) : (
-                <i
-                  className={`bx bxs-folder ${elementStyles.folderItemIcon}`}
-                />
-              )}
-              <span className={`${elementStyles.folderItemText} mr-auto`}>
-                {pageFileNameToTitle(name)}
-              </span>
-              {children ? (
-                <span className={`${elementStyles.folderItemText} mr-5`}>
-                  {children.length} item{children.length === 1 ? "" : "s"}
-                </span>
-              ) : null}
-            </div>
-          </chakra.button>
-        </LinkOverlay>
-      </LinkBox>
-      {!isDisabled && (
-        <ContextMenu>
-          <ContextMenu.Button right="1.875rem" bottom="1.875rem" />
-          <ContextMenu.List>
-            <ContextMenu.Item
-              icon={<BiEditAlt />}
-              as={RouterLink}
-              to={generatedLink}
-            >
-              <Text>Edit</Text>
-            </ContextMenu.Item>
-            <ContextMenu.Item
-              icon={<BiWrench />}
-              as={RouterLink}
-              to={
-                type === "dir"
-                  ? `${url}/editDirectorySettings/${encodedName}`
-                  : `${url}/editPageSettings/${encodedName}`
-              }
-            >
-              Settings
-            </ContextMenu.Item>
-            {type !== "dir" && (
-              <ContextMenu.Item
-                icon={<BiFolder />}
-                as={RouterLink}
-                to={`${url}/movePage/${encodedName}`}
-              >
-                <HStack spacing="4rem" alignItems="center">
-                  <Text>Move to</Text>
-                  <Icon as={BiChevronRight} fontSize="1.25rem" />
-                </HStack>
-              </ContextMenu.Item>
-            )}
-            <>
-              <Divider />
-              <ContextMenu.Item
-                icon={<BiTrash />}
-                as={RouterLink}
-                to={
-                  type === "dir"
-                    ? `${url}/deleteDirectory/${encodedName}`
-                    : `${url}/deletePage/${encodedName}`
-                }
-              >
-                Delete
-              </ContextMenu.Item>
-            </>
-          </ContextMenu.List>
-        </ContextMenu>
-      )}
+      <Box className={`${elementStyles.card} ${elementStyles.folderItem}`}>
+        <div className={contentStyles.contentContainerFolderRow}>
+          <i className={`bx bxs-file-blank ${elementStyles.folderItemIcon}`} />
+          <span className={`${elementStyles.folderItemText} mr-auto`}>
```

Comment:
The icon specified causes all items in the reordering modal to show up as pages, but we should have a separate icon for folders (`bx bxs-folder ${elementStyles.folderItemIcon}`)

Reply:
oof i derped - only saw pages so i thought we only reorder pages... good catch!